Query/Builder.php

```diff
@@ -1659,6 +1663,17 @@ public function avg($column)
     }
 
     /**
+     * Alias for the "avg" method.
+     *
+     * @param  string  $column
+     * @return float|int
```

Comment:
How can return be BOTH float and int?! PHP may lack type-enforcement, but this is just absurd.